src/status_im/ui/screens/profile/user/views.cljs

```diff
@@ -307,12 +321,76 @@
-            currency (nil? login-data)]
-           (when (nil? login-data) [advanced shown-multiaccount on-show-advanced])]]
-      [(react/safe-area-view) {:style {:flex 1}}
+           {:title               (i18n/label :t/contacts)
+            :icon                :main-icons/in-contacts
+            :accessibility-label :notifications-button
+            :accessories         [(if (pos? active-contacts-count)
+                                    (str active-contacts-count)
+                                    (i18n/label :t/none))
+                                  :chevron]
+            :on-press            #(re-frame/dispatch [:navigate-to :contacts-list])}
+           {:type :section-header
+            :container-margin-top 16
+            :title (i18n/label :t/settings)}
+           {:icon        :main-icons/security
+            :title       (i18n/label :t/privacy-and-security)
+            :accessories
+            [(when show-backup-seed?
+               [components.common/counter {:size 22} 1])
+             :chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :privacy-and-security])}
+           {:icon :main-icons/language
+            :title (i18n/label :t/language)
+            :accessories ["English" :chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :language-settings])}
+           {:icon :main-icons/notification
+            :title (i18n/label :t/notifications)
+            :accessories [:chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :notifications-settings])}
+           {:icon :main-icons/mobile
+            :title (i18n/label :t/sync-settings)
+            :accessories [:chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :sync-settings])}
+           {:icon :main-icons/settings-advanced
+            :title (i18n/label :t/advanced)
+            :accessories [:chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :advanced-settings])}
+           {:icon :main-icons/help
+            :title (i18n/label :t/need-help)
+            :accessories [:chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :help-center])}
+           {:icon :main-icons/info
+            :title (i18n/label :t/about-app)
+            :accessories [:chevron]
+            :on-press
+            #(re-frame/dispatch
+              [:navigate-to :about-app])}
+           {:icon :main-icons/log_out
+            :title (i18n/label :t/sign-out)
+            :container-margin-top 24
+            :container-margin-bottom 24
+            :theme :action-destructive
+            :on-press
+            #(re-frame/dispatch
+              [:multiaccounts.logout.ui/logout-pressed])}]]
+      ;; [my-profile-settings current-multiaccount shown-multiaccount
```

Comment:
why is it commented out? either delete or add a NOTE or TODO comment on when/why uncomment it

Reply:
Sorry, It is in my TODO I made in OP comment to this PR :). Will remove it.

Reply:
done